utils/mkhtml.py

```diff
@@ -121,6 +279,9 @@ def decode(bytes_):
   <a href="${URL_SOURCE}">${PGM} source code</a>
   (<a href="${URL_LOG}">history</a>)
 </p>
+<p>
+  Latest change: ${COMMIT_DATE} in commit: ${COMMIT}
```

Comment:
In https://github.com/ninsbl/grass/blob/e47c3673df97eaddcfa460a7a40a601b5101fc48/utils/mkhtml.py#L530-L536 I handled date from commit and file modification separately, maybe worth considering (e.g. for addons). Could be useful info to have in the manual, when it was installed... Not very important though...

Reply:
Yes I agree. I have already implemented it https://github.com/OSGeo/grass/pull/2100/commits/d3f7e81faf7dc7d6db22048a5310f3b4c3635480.